skimage/color/colorconv.py

```diff
@@ -1455,3 +1460,38 @@ def _prepare_lab_array(arr):
     if shape[-1] < 3:
         raise ValueError('Input array has less than 3 color channels')
     return dtype.img_as_float(arr, force_copy=True)
+
+
+def colormap_image(img, cmap):
+    """Draw a grayscale image by using the given colormap.
```

Comment:
"Draw" is imo just not the right term, but I'll let a native speaker comment. I would reformulate as "Colorize a grayscale image using a colormap."